index.html

```diff
@@ -0,0 +1,130 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="./css/common.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="./css/hamburger.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="./css/mobile.css">
+    <link rel="stylesheet" type="text/css" media="screen and (min-width: 747px)" href="./css/desktop.css">
+</head>
+<body>
+    <header>
+        <nav class="dk top-nav-container">
+            <article class="top-nav">
+                <ul class="social-icon">
+                    <li><img src="./assets/social-icon/facebook_f_64px.png" alt="facebook"></li>
+                    <li><img src="./assets/social-icon/twitter_64px.png" alt="twitter"></li>
+                </ul>
+                <ul class="h-link">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </article>
+        </nav>
+        <nav class="main-nav">
+            <section class="mobile-menu mb">
+                <label class="hamburger-menu mb" for="menu-checkbox">
+                    <input type="checkbox" id="menu-checkbox" />
+                </label>
+                <aside class="sidebar">
+                    <a class="new-stuff" href="#">Logout</a>
+                    <a id="about" href="./pages/about.html">About</a>
+                    <a id="program" href="#">Program</a>
+                    <a id="join" href="#">Join</a>
+                    <a id= "sponsor" href="#">Sponsor</a>
+                    <a id="news" href="#">News</a>
+                    <a class="CC" href="#">CC Campaign</a>
+                </aside>
+            </section>
+            <a href="#" class="logo dk">BLK-NG</a>
+            <article class="main-nav-link dk">
+                <a id="about" href="./pages/about.html">About</a>
+                <a id="program" href="#">Program</a>
+                <a id="join" href="#">Join</a>
+                <a id= "sponsor" href="#">Sponsor</a>
+                <a id="news" href="#">News</a>
+                <a class="CC" href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container">
+            <article class="headline">
+                <h2>"Hello! Sharing world"</h2>
+                <h2>CREATIVE COMMONS<br>GLOBAL SUBMMIT<br>2023</h2>
+                <p class="intro-txt">
+                    A joyfull celebration believing in the value of openness and sharing creating 
+                    a positive change with people from over 80 countries is atking palce in Octobar, in Lagos
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
```

Comment:
- [ ] 👍 Great job! However, the color for this paragraph does not match the design. Please update your code to use the correct color as per the design.

By ensuring that the paragraph  color aligns with the design, your project will look more visually appealing and consistent. Keep up the good work! 🎨🌟🚀